app/views/shared/_kaminari.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="govuk-grid-row pagination">
+  <div class="govuk-grid-column-one-half">
+    <div class="govuk-grid-row">
```

Comment:
This div seems to push the pages too far to the left and slightly up,  it looks fine after removing it though